api/src/main/java/org/ehcache/spi/service/PersistableResourceService.java

```diff
@@ -49,6 +50,15 @@
   Collection<ServiceConfiguration<?>> additionalConfigurationsForPool(String alias, ResourcePool pool) throws CachePersistenceException;
 
   /**
+   * Create the persistence space with the given name.
+   *
+   * @param name the name of the persistence context
+   * @param config the configuration for the associated cache
+   * @throws CachePersistenceException if the persistence space cannot be created
+   */
+  void create(String name, CacheConfiguration<?, ?> config) throws CachePersistenceException;
```

Comment:
So the addition of this method raises a number of questions:
* What is its expected usage? because there are none that I can find.
* However, it made me realise that the method `additionalConfigurationsForPool` in that same interface kind of plays that role ... amongst others. SRP violation possible there. What would be our lifecycle strategy for creating persistence spaces?